packages/affiliates/libs/synthPrices.js

```diff
@@ -0,0 +1,23 @@
+const { createReferencePriceFeedForEmp, Networker } = require("@uma/financial-templates-lib");
+const winston = require("winston");
+
+module.exports = ({ web3 } = {}) => {
+  // Fetch historic synthetic prices for a given `empAddress` between timestamps `from` and `to.
+  // Note timestamps are assumed to be js timestamps and are converted to unixtimestamps by dividing by 1000.
+  async function getHistoricSynthPrices(empAddress, from, to) {
+    from = from / 1000;
+    to = to / 1000;
+    const priceFeed = await createReferencePriceFeedForEmp(
+      winston.createLogger({ transports: [new winston.transports.Console()] }),
```

Comment:
we might want to move instantiation of this into the app or even at the "config" level and pass it down

Reply:
For sure. That makes sense. for now, I didn't want to couple anything else to Winston as I'm really keen to refactor this price feed lib away from being dependent on Winston and using the emit pattern you introduced a few PRs ago. I felt that this would be the cleanest way for now.